shellspy.go

```diff
@@ -1,11 +1,163 @@
 package shellspy
```

Comment:
1) Consider adding prebuilt binaries (GH Releases) and/or containers for various OS.  Goreleaser is a build tool that has worked for me in the past.

2) Consider adding a install.sh script to pull the correct prebuilt binary

3) Add a README page with details on how to get and utilize the library (binary, go get, etc)

Reply:
@mbarley333 goreleaser configured, can you give me an example of the install.sh script?  Have added the binaries to the .gitignore list for now did not seem right to push binaries into the git repos.